_technical/ephemeral-environments.md

```diff
@@ -1,25 +1,21 @@
 ---
 title: Ephemeral Environments
-description: Description part 1
+description: Leverage automation to create ephemeral environments on demand
 layout: pattern
 ---
 
-![Ephemeral Environments]({{ site.baseurl }}/assets/images/Generic.png)
-
-Description part 2
-
-## Context
+![Ephemeral Environments]({{ site.baseurl }}/assets/images/cs.png)
 
 A team running applications on modern cloud native platform (such as Kubernetes) often requires a variety of environments. Getting a production like environment running on a developers laptop is often not possible. The benefit of automated infrastructure means the cost of creating these environments is cheap. Running and managing them not so much.
 
-## Problem
+## In This Context
 
-Cotnainer technology has taken us a long way to resolving the 'it works on my machine' problem. However, in a microservice environment, the number of dependencies for truly validating the functionality of service can still make this a big challenge.
+Container technology has taken us a long way to resolving the 'it works on my machine' problem. However, in a microservice environment, the number of dependencies for truly validating the functionality of service can still make this a big challenge.
```

Comment:
Container technology has taken us a long way to resolve the 'it works on my machine' problem. However, in a microservice environment, the number of dependencies for truly validating the functionality of a service can still make this a big challenge.